### ISSUE TITLE
Hyperparams arguments missing

### ISSUE THREAD

**Issue by HaFred** [vonclites/squeezenet#15]
```
Squeezenet Training Program: error: the following arguments are required: --model_dir, --train_tfrecord_filepaths, --validation_tfrecord_filepaths, --network, --num_classes, --num_gpus, --batch_size, --num_input_threads, --shuffle_buffer
```
Hi, does anybody know how to set the arguments? I did not find relating files in the repo, for example the `tfrecord_filepaths`...

**Comment by vonclites** [comment]
These are command line arguments. So on the command line you would run
`python train_squeezenet.py --model_dir=<path_to_model_dir>` etc.
All the parameters are specified in https://github.com/vonclites/squeezenet/blob/master/arg_parsing.py
However, this was implemented using a very old version of Tensorflow... the API it uses might be deprecated by now. You may be better off using it as a guideline for a reimplementation in Tensorflow 2.

**Comment by vonclites** [comment]
Also, the tfrecords aren't provided in this repository. You would have to create them. See https://github.com/vonclites/squeezenet/blob/f13555462e835409480829f8fdacac96835161e5/inputs.py#L169 for the fields the tfrecord examples are expected to have (an 'image' represented as a byte string and an int64 label).
